transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/AddTransferMethodTest.java

```diff
@@ -73,13 +91,25 @@ protected Intent getActivityIntent() {
                     intent.putExtra("TRANSFER_METHOD_TYPE", "BANK_ACCOUNT");
                     intent.putExtra("TRANSFER_METHOD_COUNTRY", "US");
                     intent.putExtra("TRANSFER_METHOD_CURRENCY", "USD");
+                    intent.putExtra("TRANSFER_METHOD_PROFILE_TYPE", "INDIVIDUAL");
                     return intent;
                 }
             };
 
+    @Captor
+    ArgumentCaptor<String> mPageNameCaptor;
+    @Captor
+    ArgumentCaptor<String> mPageGroupCaptor;
+    @Captor
+    ArgumentCaptor<ErrorInfo> mErrorInfoCaptor;
+
+    @Mock
+    private HyperwalletInsight mHyperwalletInsight;
```

Comment:
Using HyperwalletInsight as mock with HyperwalletInsight.setInstance(mHyperwalletInsight); is introducing "false memory leaks" to the tests. In order to keep those tests memory leak-free, please apply the following changes to this and the other tests in this pull request:
Removing the annotation @Mock from the HyperwalletInsight declaration, creating the mock for each test and cleaning it up after each test. 


```suggestion
private HyperwalletInsight mHyperwalletInsight;

    @Before
    public void setup() {
        Hyperwallet.getInstance(new TestAuthenticationProvider());
        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
                .getResourceContent("authentication_token_response.json")).mock();
        mHyperwalletInsight = mock(HyperwalletInsight.class);
        HyperwalletInsight.setInstance(mHyperwalletInsight);
    }

    @After
    public void cleanup() {
        TransferMethodRepositoryFactory.clearInstance();
        mHyperwalletInsight = null;
        HyperwalletInsight.setInstance(null);
    }
```